recipe-server/normandy/recipes/templates/identicon.svg

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" width="128" height="128">
+    <g>
+        <defs>
+            <clipPath id="shield-border" transform="scale(100)">
+                <path d="M 0.5 0.0
+                         C 0.6 0.1, 0.7 0.15, 0.9 0.15
+                         C 0.9 0.5, 0.9 0.8, 0.5 1.0
+                         C 0.1 0.8, 0.1 0.5, 0.1 0.15
+                         C 0.3 0.15, 0.4 0.1, 0.5 0.0"/>
+            </clipPath>
+        </defs>
+        <g clip-path="url(#shield-border)">
+            <rect fill="{{  field_color }}" width="100" height="100"/>
+            {% if treatment == 'TwoColor' %}
```

Comment:
Above I was assuming that the treatment was generating this XML and the template didn't need to know which treatment was being used, and I think that's a cleaner design anyway.

Reply:
As discussed in person, we decided that since there are only two treatments, it's better to leave the template as is.